src/Instagram/Provider.php

```diff
@@ -44,24 +53,19 @@ protected function getTokenUrl()
      */
     protected function getUserByToken($token)
     {
-        $endpoint = '/users/self';
-        $query = [
-            'access_token' => $token,
-        ];
-        $signature = $this->generateSignature($endpoint, $query);
-
-        $query['sig'] = $signature;
-        $response = $this->getHttpClient()->get(
-            'https://api.instagram.com/v1/users/self',
-            [
-                RequestOptions::QUERY   => $query,
-                RequestOptions::HEADERS => [
-                    'Accept' => 'application/json',
-                ],
-            ]
-        );
+        $meUrl = 'https://graph.instagram.com/me?access_token='.$token.'&fields='.implode(',', $this->fields);
```

Comment:
Can you link to the docs? this look a major change in API

Reply:
https://developers.facebook.com/docs/instagram-basic-display-api/reference/me

https://developers.facebook.com/docs/instagram-basic-display-api/reference/user

https://developers.facebook.com/docs/instagram-basic-display-api/reference/oauth-authorize

Reply:
Meta is trying to discourage people from using login with IG and directing them to use FB. So the data is limited. the previous endpoint of `/users/self` no longer exists and will 404. I believe they replaced it with `/me`.